### DISCUSSION TITLE
What about a very quick and very dirty PenpotJam ?

### DISCUSSION THREAD

**Discussion by mlgarchery** [uxbox/uxbox#1368] [Ideas]
Hello guys,
you probably know the FigJam feature from Figma.  What about something similar for Penpot ?
I know there is a lot to do already, but why not think about the future for a moment.
We already have the tools to create PenpotJam kind of project, which would be a simplified penpot project, focused on ease of use.
No artboard, or an infinite one. Simple geometrical forms (square, circle) containing text, with arrows connecting them. That's all.
This way penpot could help teams, and particularly people not in the design industry, to express their thought with the same tool.

Penpot is a great collaboration tool.
I hope to see it grow in the future !

**Comment by hirunatan** [comment]
This is a great idea. At Penpot we already know that the app may be used for live collaboration sessions (see the whiteboarding kit at https://penpot.app/libraries-templates.html). It's likely that we'll implement something like you are asking. Keep following our backlog.

Thanks for your suggestions.

**Comment by parkerjm** [comment]
With Adobe purchasing Figma, I hope this comes to fruition! We love FigJam but don't want to support Adobe.

**Comment by altosaar** [comment]
Seconding this, we need this @onefact!!

**Comment by lessless** [comment]
That would make a big incentive to adopt Penpot because it will push value across multiple dimensions. 
Including consolidation of products, i.e. Miro + Figma -> Penpot (that's a lot of saved budget).

**Reply by achrllrogia45** [reply]
FR FR... smh 😌

**Comment by achrllrogia45** [comment]
AAAaaaa.... I want PenpotJam so much 😭😭, this will so great

**Comment by DesertBanana** [comment]
Does this exist yet?
